specs/pvs/corla.pvs

```diff
@@ -6,7 +6,18 @@
 %%%
 %%% \section{Introduction}
 %%% This document contains a formal specification of the ColoradoRLA
-%%% system/tool. It is written in a literate style using the PVS
+%%% system/tool. 
+%%% It contains a detailed technical specification of the system,
+%%% annotated with prose designed to help readers who are not expert
+%%% in formal system specification understand the scope and interpretations
+%%% of the elements of the specification. The details of the specification
+%%% itself will guide deployment, management, maintenance and evolution 
+%%% of the system delivered to the Colorado Department of State. The prose
+%%% annotations should help Colorado Department of State elections domain experts
+%%% and technical staff to assess the completeness and correctness of the 
+%%% ColoradoRLA system/tool.
+%%% 
+%%% This document is written in a literate style using the PVS
 %%% theorem prover. Writing system specifications in this fashion is
 %%% called (formal) domain engineering. Our formal domain model is
 %%% written in PVS's higher-order logic (HOL). Our informal domain
```

Comment:
Good catch. Note that since this is a comment on text that isn't near a revision from this PR, we can't easily link the comment to the line.  And since the line number can change with future commits, here is some more context, lines 163-166:

```
%%% values. Email is used for communication between officials and
%%% with the ?. Files are used to store information in a
%%% persistent fashion. Consequently, we introduce a few types for
%%% these ideas.
```